packages/create-pantheon-decoupled-kit/src/actions/convertCSSModules.ts

```diff
@@ -0,0 +1,71 @@
+import chalk from 'chalk';
+import { execSync } from 'child_process';
+import { Action, isString } from '../types';
+import autoprefixer from 'autoprefixer';
+import postcss from 'postcss';
+import fs from 'fs-extra';
+import klaw from 'klaw';
+import globalData from '@csstools/postcss-global-data';
+import customProperties from 'postcss-custom-properties';
+import { rootDir } from '..';
+
+export const getAllFiles = async (
+	templateDir: string,
+	outDir: string,
+): Promise<string[]> => {
+	const templates: string[] = [];
+
+	// loop through each dir
+	for await (const file of klaw(templateDir)) {
+		if (file.stats.isDirectory()) continue;
+		const templateName = file.path.split(outDir)[1];
+
+		templates.push(templateName);
+	}
+	return templates;
+};
+
+export const convertCSSModules: Action = async ({ data }) => {
+	if (data?.noInstall || !data?.tailwindcss)
+		return 'skipping CSS module conversion';
+	if (!data.outDir || !isString(data?.outDir))
+		throw new Error('outDir is not valid');
+	data.silent || console.log(chalk.green('Converting Modules...'));
```

Comment:
```suggestion
	data.silent || console.log(chalk.green('Converting CSS modules to tailwindcss classes...'));
```
"Modules" by itself is a bit vague.